pyproject.toml

```diff
@@ -20,5 +20,11 @@ pre-commit = "2.20.*"
 [tool.poetry.group.test.dependencies]
 pytest = "7.1.*"
 
+[tool.poetry.group.frontend.dependencies]
+fastapi = "0.95.*"
+jinja2 = "3.1.*"
+python-multipart="0.0.*"
+starlette = "0.26.*"
+uvicorn = "0.22.*"
 
 # poetry export --without-hashes --format=requirements.txt --with="style,test" --output=requirements.txt
```

Comment:
Może warto byłoby to dodać do Make'a, np. do `make install`, które sugerowałem wyżej? Wtedy mamy gwarancję, że `requirements.txt` istnieje i jest zgodne z najnowszymi zależnościami